projects/fal/src/fal/api/api.py

```diff
@@ -1544,15 +1536,3 @@ def on(  # type: ignore[no-overload-impl]
     def on(
         self, host: Host | None = None, *, serve: Literal[False], **config: Any
     ) -> IsolatedFunction[ArgsT, ReturnT]: ...
-
-
-class Server(uvicorn.Server):
-    """Server is a uvicorn.Server that actually plays nicely with signals.
-    By default, uvicorn's Server class overwrites the signal handler for SIGINT,
-    swallowing the signal and preventing other tasks from cancelling.
-    This class allows the task to be gracefully cancelled using asyncio's built-in task
-    cancellation or with an event, like aiohttp.
-    """
-
-    def install_signal_handlers(self) -> None:
```

Comment:
no longer necessary?

Reply:
no longer necessary, because we now actually want uvicorn to catch the signal. plus, uvicorn.Server no longer have `install_signal_handlers` anyway.